internal/stack/boot.go

```diff
@@ -41,6 +41,8 @@ func BootUp(options Options) error {
 		if err != nil {
 			return errors.Wrap(err, "copying package contents failed")
 		}
+	} else {
+		fmt.Println("WARNING: no custom build packages directory found. Any locally built integrations will not be installed.")
```

Comment:
There is a message logged in L39 informing about used package directories. Do you see this as insufficient/less informative/improvement needed?

Reply:
Yah, I think we need something that prints a message in the negative case and actually reports when something isn't working as expected.

Reply:
It is working as expected as the process wasn't started in the integrations repo. Maybe we should rephrase this warning into something like:

`logger.Info("Local packages won't be included as the elastic-package didn't find any built packages.")`.

WDYT?